NitroxServer/NitroxServer.csproj

```diff
@@ -3,6 +3,7 @@
     <PropertyGroup>
         <TargetFramework>net472</TargetFramework>
         <Nullable>disable</Nullable>
+        <Configurations>Debug;Release;LinuxRelease</Configurations>
```

Comment:
This changes can be safely removed without changing behavior.

Reply:
Actually LinuxRelease is configured to not build the NitroxLauncher and removing it will actually drop the ability to be able to compile Nitrox on Linux in its current state, granted Nitrox cannot run on Linux as it stands right now...
![image](https://github.com/SubnauticaNitrox/Nitrox/assets/13115936/da040834-2f4e-4c3d-a17a-b0c641ac8f13)

Reply:
It compiles fine on windows when I removed the line. Did you confirm this behavior after you removed the line?

I ask because `Configuration` is set by an argument to the MSBuild CLI (called by the IDE), depending on which you selected in the IDE drop down. It should not be set by .props/.targets but can be used in conditionals to do something different (like optimize code in RELEASE config).